cull_secrets/clean_user_registry_secrets.py

```diff
@@ -33,12 +33,13 @@
 POD_LABELS = [  # unlikely to have invalid characters, used to quickly filter
     "renku.io/commit-sha",
     "renku.io/username",
-    "renku.io/projectName",
+    "renku.io/projectId",
```

Comment:
Maybe we can use gitlabProjectId here (and in other corresponding locations in the code) just to be extra clear?

Reply:
Sure I will change the name.